### ISSUE TITLE
Changing the image in login page

### ISSUE THREAD

**Issue by AswaniBolisetti** [samyakmaitre/eventmint#320]
@samyakmaitre 
In the login page, the image quality is not good, I would like to enhance it
could you please assign this to me

**Comment by samyakmaitre** [comment]
Hi there,

Thank you for your interest! This issue is not required at the moment. However, if you can enhance the entire login page UI, you are more than welcome to do so. Please create a new issue for it, and feel free to sync your fork before starting. I'd be happy to review your work once it's submitted.

Best regards,
Samyak
